mcworldlib/anvil.py

```diff
@@ -394,6 +407,54 @@ def uncache(self, pos: u.RegionPos, recursive: bool = False):
         self._items[pos] = candidates[0]
 
 
+def compress_lz4_java(data: bytes, compression_level: int = 6) -> bytes:
+    """
+    Compress data with LZ4 and lz4-java header
+    (Compression level 6 is default for vanilla minecraft)
+    """
+    token = 0x20 | compression_level
```

Comment:
`0x20` is another candidate for a module-level constant (`COMPRESSION_TYPE_LZ4` perhaps?). Actually in LZ4-Java this could be also be `0x10` for no compression (`COMPRESSION_TYPE_RAW`?), which would happen as a fallback if the compressed data is larger than the original data.

Reply:
I implemented Raw compression